src/main/java/com/yeolsimee/moneysaving/app/user/entity/User.java

```diff
@@ -32,14 +34,24 @@ public class User extends BaseEntity implements UserDetails  {
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "user_id")
     private long id;
```

Comment:
```suggestion
    private Long id;
```
Entity 클래스에서 필드값은 null값 체크를 위해서 primitive type이 아닌 wrapper type으로 지정하는 좋을 것 같습니다~!

https://velog.io/@d-h-k/JPA-Entity-Class-%EC%97%90%EC%84%9C-Primitive-Type-%EC%9D%84-%EC%8D%A8%EC%95%BC%ED%95%A0%EA%B9%8C-Wrapper-Class-%EB%A5%BC-%EC%82%AC%EC%9A%A9%ED%95%B4%EC%95%BC%ED%95%A0%EA%B9%8C